azure/appservice/main.tf

```diff
@@ -0,0 +1,61 @@
+resource "azurerm_resource_group" "app" {
+  name     = "rg-${var.project}"
+  location = local.location
+
+  tags = {
+    country     = var.country
+    environment = var.environment
+    squad       = var.squad
+  }
+}
+
+resource "azurerm_app_service_plan" "app" {
+  name                = "asp-${var.project}"
+  location            = azurerm_resource_group.app.location
+  resource_group_name = azurerm_resource_group.app.name
+  kind                = "Linux"
```

Comment:
We should test this to see if Nmbrs runs on these Linux app services. Even though you're using .net version of it, I'm not sure there aren't issues with it because they point some files to windows directories